python/imjoy_rpc/rpc.py

```diff
@@ -481,9 +495,9 @@ def _handle_callback(self, data):
                 self._run_with_context(
                     self._call_method, method, *args, method_name=data["id"]
                 )
-        except Exception as e:
+        except Exception as err:
             traceback_error = traceback.format_exc()
-            logger.error("error when calling callback function: %s", traceback_error)
+            logger.exception("error when calling callback function: %s", err)
```

Comment:
Curious about what is the difference between the two?

We will get a stack of traceback with logger.exception?